firebase_connector/models/firebase_notification.py

```diff
@@ -142,6 +147,11 @@ def send(self, **kwargs):
                     [("partner_id", "=", False)]
                 )
 
+            # filter registration based on language
+            if notif.language:
+                registration_ids = registration_ids.filtered(
+                    lambda reg: not reg.language or reg.language == notif.language)
+
```

Comment:
Why not adding the filter directly in the search method above at lines 147 and 143?

Reply:
Simply because there is less repetition and I also find it a bit clearer when handling notification with no language specified (for example old notification still running when the changes will be published). Same goes for handling registration with no set language. 

But I am open to the idea of doing it differently